drivers/hmis/lib/hmis_util/json_forms.rb

```diff
@@ -26,10 +26,14 @@ def seed_all
     protected
 
     def env_key
-      return ENV['CLIENT'] if ENV['CLIENT'].present?
-
-      # default to QA environment in development to get forms with all possible questions enabled
-      'qa_hmis' if Rails.env.development?
+      @env_key ||= if Rails.env.test?
```

Comment:
This seems like a very inexpensive method. Is memoization worth the bother?

Reply:
not really but I had my print statements in there and I was bothered by the repetition.. I was having trouble getting `CLIENT=test` to be consistently set in the test environment, which is why I switched it to check `Rails.env.test?` instead